mllib/src/main/scala/org/apache/spark/ml/regression/LinearRegression.scala

```diff
@@ -85,6 +85,18 @@ class LinearRegression(override val uid: String)
   setDefault(fitIntercept -> true)
 
   /**
+   * Whether to standardize the training features before fitting the model.
+   * The coefficients of models will be always returned on the original scale,
+   * so it will be transparent for users. Note that when no regularization,
```

Comment:
probably s/when no/without/ or s/when no regularization/when no regularization is applied/  is a bit easier to read (but its a minor nit so no stress).